dhis-2/dhis-api/src/main/java/org/hisp/dhis/icon/CustomIcon.java

```diff
@@ -49,4 +49,36 @@ public class CustomIcon implements Icon {
   private String fileResourceUid;
 
   private String createdByUserUid;
+
+  private boolean custom;
+
+  private Date created;
+
+  private Date lastUpdated;
+
+  public CustomIcon(
```

Comment:
`custom` shouldn't be part of the constructor and `isCustom` should always return true